ui/src/app/layout/applications/application-block.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="card" *ngIf="environmentsService.getCurrentEnvironment() | async; let env">
-    <div class="card-header bg-warning">
+    <div class="card-header bg-topicSettings">
```

Comment:
I'm not sure I like this approach much - I think we should stick with the standard bootstrap classes whereever possible, and only change their values when switching to dark mode. This would give a more consistent design.